Fixed float conversion build errors

randallli opened the pull request:
Implicit conversion loses floating-point precision: 'double' to 'CGFloat' (aka 'float')

Found by turning on warnings via: https://github.com/material-components/material-components-ios/pull/2480

randallli committed:
Fixed float conversion build errors
Implicit conversion loses floating-point precision: 'double' to 'CGFloat' (aka 'float')

Found by turning on warnings via: https://github.com/material-components/material-components-ios/pull/2480

samnm commented on components/Collections/examples/CollectionsInkExample.m:
```diff
@@ -73,7 +73,7 @@ - (UIColor *)collectionView:(UICollectionView *)collectionView
     return [MDCPalette.lightBluePalette.tint500 colorWithAlphaComponent:0.2f];
   } else if (indexPath.item == 2) {
-    return [UIColor colorWithRed:1.0 green:0.0 blue:0.0 alpha:0.2];
+    return [UIColor colorWithRed:1.0 green:0.0 blue:0.0 alpha:0.2f];
```
Are the 1.0 and 0.0 values here ok?

randallli replied:
Done.
very odd. the compiler didn't complain about these.

randallli committed:
More Fixed float conversion build errors
Implicit conversion loses floating-point precision: 'double' to 'CGFloat' (aka 'float')

Found by turning on warnings via: https://github.com/material-components/material-components-ios/pull/2480

ianegordon commented:
Opened issue #2605 to track the float casting in our wider codebase.
